internal/driver/glfw/canvas.go

```diff
@@ -160,7 +161,7 @@ func (c *glCanvas) Resize(size fyne.Size) {
 			// TODO: remove this when #707 is being addressed.
 			// “Notifies” the PopUp of the canvas size change.
 			p.Resize(p.Content.Size().Add(fyne.NewSize(theme.Padding()*2, theme.Padding()*2)))
-		} else if p, ok := overlay.(*widget.TickerPopUp); ok {
+		} else if p, ok := overlay.(*dialog.TickerPopUp); ok {
```

Comment:
This special case should not be required, the size management should be possible within the code.
See note above how special cases are being removed

Reply:
> This special case should not be required, the size management should be possible within the code.
> See note above how special cases are being removed
I'd love to be able to remove this but haven't figured out how yet.  So for now it stays.  I'll try to figure it out before I publish this pr.

Reply:
Still haven't found way to deal with this elegantly.